.changeset/strange-bats-listen.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sumup/icons': minor
+---
+
+Added the `IconsManifest` type to the exports. It describes the shape of `@sumup/icons/manifest.json`.
```

Comment:
Surprising to find here but a nice idea 👌😛

Reply:
I needed it for the Storybook components 🤷‍♂️